OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -786,10 +787,33 @@ public void ResolveOrder(Actor self, Order order)
 				else
 					self.QueueActivity(new HeliReturnToBase(self, Info.AbortOnResupply, null, false));
 			}
+			else if (order.OrderString == "Scatter")
+				MoveToRandomCell(self);
 		}
 
 		#endregion
 
+		void MoveToRandomCell(Actor self)
+		{
+			var currCell = self.Location;
```

Comment:
All this can simplify down to
```csharp
var offset = new WVec(0, -self.World.SharedRandom.Next(512, 2048), 0)
	.Rotate(WRot.FromFacing(self.World.SharedRandom.Next(256)));
var target = Target.FromPos(self.CenterPosition + offset);

self.CancelActivity();
self.SetTargetLine(target, Color.Green, false);
if (!Info.CanHover)
```

I turned down the scatter range a bit too, it felt far too large IMO before.

Reply:
Done.